app/controllers/api/internal/session_comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Internal::SessionCommentsController < Api::Internal::BaseController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 17